playbooks/common/openshift-cluster/config.yml

```diff
@@ -45,6 +45,10 @@
   tags:
   - hosted
 
+- include: service_catalog.yml
```

Comment:
Yeah that's fine, it's easy to move around until we come out of tech preview anyway.
```
  when:
  - openshift_enable_service_catalog | default(false)
```